WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -2975,6 +2990,9 @@ This part is the link to the website, and forms part of a longer sentence which
 /* Display label for the product's pending status */
 "Pending review" = "Pending review";
 
+/* Title for the amount field on the Fee Details screen during order creationParameters: %1$@ - percent sign */
+"Percentage (%1$@)" = "Percentage (%1$@)";
```

Comment:
Found in the version with the new tooling, too. See [`5c21528`](https://github.com/woocommerce/woocommerce-ios/pull/6737/commits/5c215284d6839f397732f684ec664405f42939ab#diff-96d03942a96e588fcdca0288a2a18d0466a6e44ee5b256f581cd17f2086ea22dR3133-R3134)